### PR TITLE
New package: lastpass-cli-0.9.0

### PR DESCRIPTION
**Pull request by jaagr**
Package for command line interface to LastPass.com

### CODE REVIEW

File: srcpkgs/lastpass-cli/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+makedepends="asciidoc libressl-devel libcurl-devel libxml2-devel"
+depends="libressl curl libxml2"
+short_desc="Command line interface to LastPass.com"
+maintainer="Michael Carlberg <c@rlberg.se>"
+license="GPL-2"
+homepage="https://lastpass.com/"
+distfiles="https://github.com/lastpass/lastpass-cli/archive/v${version}.tar.gz"
+checksum=e7314f0dfeec86add7c19c053ee34bb7a176e462e71727d2b345481d2d136800
+
+do_build() {
```
Comment (Vaelatern) [inline srcpkgs/lastpass-cli/template]: `build_style=gnu-makefile`
`make_build_target="all doc"
`make_install_target="install install-doc"`

Comment (Vaelatern) [inline srcpkgs/lastpass-cli/template]: This whole function is unecessary.

File: srcpkgs/lastpass-cli/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	cd $wrksrc
+	make all doc
+}
+
+do_install() {
+	cd $wrksrc
```
Comment (Vaelatern) [inline srcpkgs/lastpass-cli/template]: No need for `cd $wrksrc`

File: srcpkgs/lastpass-cli/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'lastpass-cli'
+pkgname=lastpass-cli
+version=0.9.0
+revision=1
+makedepends="asciidoc libressl-devel libcurl-devel libxml2-devel"
```
Comment (Vaelatern) [inline srcpkgs/lastpass-cli/template]: `asciidoc` should be in `hostmakedepends`

File: srcpkgs/lastpass-cli/template [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+depends="libressl curl libxml2"
```
Comment (Vaelatern) [inline srcpkgs/lastpass-cli/template]: `depends="curl libxml2"`

Comment (dominikh) [inline srcpkgs/lastpass-cli/template]: Why are curl and libxml2 in depends? Unless it's actually shelling out to the curl binary, curl isn't needed. And unless it is using one of the utilities in libxml2, that's not needed, either. Library dependencies get detected automatically.

File: srcpkgs/lastpass-cli/template (hunk 1 above)
Comment (ebfe) [inline srcpkgs/lastpass-cli/template]: Not entirely, with the `gnu-makefile` do_build it requires a Makefile patch, so not sure if this is an improvement.

File: srcpkgs/lastpass-cli/template [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+homepage="https://lastpass.com/"
+distfiles="https://github.com/lastpass/lastpass-cli/archive/v${version}.tar.gz"
+checksum=e7314f0dfeec86add7c19c053ee34bb7a176e462e71727d2b345481d2d136800
+
+pre_build() {
+	sed -i 's/^CFLAGS +=/override CFLAGS +=/g' Makefile
+	sed -i 's/^LDLIBS +=/override LDLIBS +=/g' Makefile
+}
+
+do_install() {
```
Comment (Vaelatern) [inline srcpkgs/lastpass-cli/template]: If you change this to `post_install` then you can remove the `make` line, the `vbin` line, and the `vman` line.

**Pull request merged**